pages/api/v1/answers.ts

```diff
@@ -474,6 +494,13 @@ async function Angela( // Good ol' Angela
       totalCost += outCost;
 
       mostRecentParsedOutput = parseOutput(rawOutput);
+
+      if (
+        mostRecentParsedOutput.commands &&
+        mostRecentParsedOutput.commands.length > 0
+      )
+        numUserQueries += 1;
```

Comment:
Since we've discussed around 237 different ways of counting queries, please could you write a comment documenting which we've actually gone for and how this implements that?

Just I can totally imagine forgetting which we went with in 5 days (or 5 minutes!)